### PR TITLE
test(Tooltip): add test cases for all props

### PR DESCRIPTION
**Pull request by fabriciomendonca**

### CODE REVIEW

File: src/components/Tooltip/Tooltip.cy.tsx [hunk 1]
```diff
@@ -191,4 +198,111 @@ describe('Tooltip Component', () => {
         cy.get(TOOLTIP_ID).should('exist');
         cy.get(TOOLTIP_ID).should('have.class', 'tw-hidden');
     });
```
Comment (dominiquemb) [inline src/components/Tooltip/Tooltip.cy.tsx:200]: I think there should be a test added for the content prop itself.

File: src/components/Tooltip/Tooltip.cy.tsx [hunk 2]
```diff
@@ -191,4 +198,111 @@ describe('Tooltip Component', () => {
...
+
```
Comment (dominiquemb) [inline src/components/Tooltip/Tooltip.cy.tsx:201]: Also missing test a case for triggerElement

**Review by dominiquemb — CHANGES_REQUESTED** [review]
See comments inline

File: src/components/Tooltip/Tooltip.cy.tsx (hunk 1 above)
Comment (fabriciomendonca) [reply src/components/Tooltip/Tooltip.cy.tsx:200]: The first test case has only the `content` prop in it, so it is already there. I will update the title to `should render a tooltip with content`.
BTW, `content` is the only required prop.

**Review by dominiquemb — APPROVED** [review]

**Pull request merged**